### PR TITLE
APIGOV-21779 Asset Catalog content

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Added new section

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/manage_asset_catalog/_index.md [hunk 1]
```diff
@@ -2,5 +2,25 @@
...
+
+* Are similar in nature but running on different data planes. For example, Petstore API 1.0 in Axway API Gateway and Petstore API 1.0 in Azure Gateway can be grouped in a Petstore API.
+
+* Have a functionality that makes them complimentary. For example, Employee API, HR Administration API, and PayRoll API can be grouped in an HR Asset.
+
+* Are similar in nature but have been adapted for a specific geography. For example, Payment Europe API and Payments US API can be grouped in a Payments API.
+
+The asset model is flexible, so APIs can be group to suit individual needs.
+
+Once the Asset Catalog has been used to manage and group API Services into assets, they are deploy as products in the [Product Catalog](/docs/manage_product_catalog) where they can be attached to subscription plans and consumed from the [Marketplace](/docs/manage_marketplace).
```
Comment (cbordier) [inline content/en/docs/manage_asset_catalog/_index.md:24]: Replace Product Catalog by Product Foundry?

File: content/en/docs/manage_asset_catalog/asset_management.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+## View assets
+
+To view assets:
+
+1. Navigate to *Asset Catalog*.
+2. Select **Assets > All Assets**.
+
+View the following information for all assets in any state (Draft, Active, Deprecated, Active):
+
+* Name - Uniquely identifies the asset
```
Comment (cbordier) [inline content/en/docs/manage_asset_catalog/asset_management.md:18]: @robertengaxway : Name and Title are not the values displayed in UI. Asset Name / Logical Name
Maybe something to align.

File: content/en/docs/manage_asset_catalog/_index.md (hunk 1 above)
Comment (lbadenhop) [reply content/en/docs/manage_asset_catalog/_index.md:24]: Yes, I need to change all instances of Product Catalog/catalog to Product Foundry.

**Comment by lbadenhop** [comment]
@cbordier @robertengaxway Please provide the missing content. Thanks!

**Comment by lbadenhop** [comment]
@robertengaxway @cbordier Integrate with API / CLI topic created.

**Comment by lbadenhop** [comment]
@DesertLeprechaun Can you please provide content for the "Integrate with API / CLI" topic in this pull request? If you aren't able to provide the content, please let me know. Thanks!

**Comment by lbadenhop** [comment]
@robertengaxway Please provide the missing content in "Asset Management" topic. I think we can push this to pre-prod without the "Tags, attributes' Categories" content. content/en/docs/integrate_with_central/cli_central/build_an_asset_using_the_axway_central_cli.md was set to "draft" to remove it from being published. Dave's API/CLI content covers the content in this older topic.

**Comment by lbadenhop** [comment]
@cbordier @robertengaxway I think the asset_management and asset_integrate_api_cli topics need to be aligned. Dave included CLI content for processes that aren't documented in the asset_managment topic.

**Comment by lbadenhop** [comment]
@robertengaxway Please mark as reviewed so I can merge. Thanks.

**Review by robertengaxway — APPROVED** [review]
I approve of the content

**Pull request merged**